client/src/pages/profile/profile.js

```diff
@@ -0,0 +1,139 @@
+import { connect, useDispatch } from "react-redux";
+import { useParams } from "react-router";
+import { useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import {
+  fetchUser,
+  fetchFollowers,
+  fetchFollowing,
+  followUser,
+  unfollowUser,
+} from "actions";
+import AppLayout from "components/layouts/AppLayout";
+import Loading from "components/layouts/Loading";
+
+const Profile = (props) => {
+  let { userId } = useParams();
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
```

Comment:
You are already using dispatch here, better to use this instead of "props.Action"

Reply:
Done.